src/hdmf/backends/io.py

```diff
@@ -75,6 +75,15 @@ def read(self, **kwargs):
         return container
 
     @docval({'name': 'container', 'type': Container, 'doc': 'the Container object to write'},
+            {
```

Comment:
@CodyCBakerPhD can you move this to hdmf-zarr?

With `docval(..., allow_extra=True)` and `**kwargs` being passed around, it will make its way to `write_builder`

https://github.com/hdmf-dev/hdmf-zarr/blob/d47fc8249e76a371fff990b07218a088e59ea590/src/hdmf_zarr/backend.py#L200

Reply:
Since this arg is specific to Zarr at the moment, I think it makes the most sense to move it to hdmf_zarr

Reply:
Ahhh, because of `allow_extra=True`, I see